### PR TITLE
network: Add SUBMENU variable for some packages

### PR DESCRIPTION
**Pull request by Ra2-IFV**
Add SUBMENU variable for some packages in Network category. Add title for `dnscrypt-proxy` package.

Filesystem:
- cifs-utils

File Transfer:
- onionshare-cli

IP Addresses and Names:
- dnscrypt-proxy2
- family-dns
- https-dns-proxy
- mdns-repeater
- nextdns
- smartdns

Version Control Systems:
- git-lfs

Web Servers/Proxies:
- xray-core

Wireless:
- dawn

Maintainer: Should I ping all maintainers for these packages?
Compile tested: no, simple commit
Run tested: no

Description:

### CODE REVIEW

File: net/xray-core/Makefile [hunk 1]
```diff
@@ -30,6 +30,7 @@ define Package/xray/template
   TITLE:=A platform for building proxies to bypass network restrictions
   SECTION:=net
   CATEGORY:=Network
+  SUBMENU:=Web Servers/Proxies
```
Comment (1715173329) [inline net/xray-core/Makefile:33]: The same as https://github.com/openwrt/packages/pull/25391#discussion_r1845437256.

**Comment by 1715173329** [comment]
Please do not add me to `Co-authored-by`.

**Comment by Ra2-IFV** [comment]
> Please do not add me to `Co-authored-by`.

Okay sorry, I'm trying to follow some rules given by other developers...

File: net/dnscrypt-proxy/Makefile [hunk 2]
```diff
@@ -32,6 +32,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/dnscrypt-proxy/Default
   SECTION:=net
   CATEGORY:=Network
+  TITLE:=DNSCrypt-Proxy by Dyne, not actively maintained
```
Comment (1715173329) [inline net/dnscrypt-proxy/Makefile:35]: ideally title should be put below submenu.

Comment (Ra2-IFV) [reply net/dnscrypt-proxy/Makefile]: this variable is missing in the wiki, hope I could apply for an account
https://openwrt.org/docs/guide-developer/packages#buildpackage_defines

**Pull request merged**